Plugin/FakeIpPlugin.php

```diff
@@ -76,6 +76,9 @@ public function handleQuery(Query $query, callable $next, callable $first)
         if ($count > 0) {
             $query = $query->withText($text);
         }
+        if (null === $this->needle || '' === $this->needle) {
+            $query = $query->withText($this->replacement);
```

Comment:
```suggestion
            $query = $query->withText($replacement);
```
because it could be generated from faker

Reply:
oups sorry, I will update this